docs/sources/reference/commandline/cli.md

```diff
@@ -48,6 +48,34 @@ These Go environment variables are case-insensitive. See the
 [Go specification](http://golang.org/pkg/net/http/) for details on these
 variables.
 
+## Configuration Files
+The Docker command line stores its configuration files in a directory
```

Comment:
## Configuration Files

The Docker command line stores its configuration files in a directory called
`.docker` within your `HOME` directory. Docker manages most of the files in
`.docker` and you should not modify them. However, you *can modify* the
`.docker/config.json` file to control certain aspects of how the `docker`
command behaves.

Currently, you can modify the `docker` command behavior using environment variables or command-line options. You can also use options within `config.json` to modify that same behavior.  When using these mechanisms, you must keep in mind the order of precedence among them. Command line options override environment variables and environment variables override properties you specify in a `config.json` file.

The `config.json` file stores a JSON encoding of a single `HttpHeaders`
property. The property specifies a set of headers to include in all
messages sent from the Docker client to the daemon. Docker does not try to
interpret or understand these header; it simply puts them into the messages.
Docker does not allow these headers to change any headings it sets for itself.

Following is a sample `config.json` file:

    {
      "HttpHeaders: {
        "MyHeader": "MyValue"
      }
    }